ratatool-sampling/src/main/scala/com/spotify/ratatool/samplers/BigSampler.scala

```diff
@@ -101,16 +101,18 @@ object BigSampler extends Command {
     println(s"""BigSampler - a tool for big data sampling
         |Usage: ratatool $command [dataflow_options] [options]
         |
-        |  --sample=<percentage>                      Percentage of records to take in sample, a decimal between 0.0 and 1.0
-        |  --input=<path>                             Input file path or BigQuery table
-        |  --output=<path>                            Output file path or BigQuery table
-        |  [--fields=<field1,field2,...>]             An optional list of fields to include in hashing for sampling cohort selection
-        |  [--seed=<seed>]                            An optional seed used in hashing for sampling cohort selection
-        |  [--hashAlgorithm=(murmur|farm)]            An optional arg to select the hashing algorithm for sampling cohort selection. Defaults to FarmHash for BigQuery compatibility
-        |  [--distribution=(uniform|stratified)]      An optional arg to sample for a stratified or uniform distribution. Must provide `distributionFields`
-        |  [--distributionFields=<field1,field2,...>] An optional list of fields to sample for distribution. Must provide `distribution`
-        |  [--exact]                                  An optional arg for higher precision distribution sampling.
-        |  [--byteEncoding=(raw|hex|base64)]          An optional arg for how to encode fields of type bytes: raw bytes, hex encoded string, or base64 encoded string. Default is to hash raw bytes.
+        |  --sample=<percentage>                               Percentage of records to take in sample, a decimal between 0.0 and 1.0
+        |  --input=<path>                                      Input file path or BigQuery table
+        |  --output=<path>                                     Output file path or BigQuery table
+        |  [--fields=<field1,field2,...>]                      An optional list of fields to include in hashing for sampling cohort selection
+        |  [--seed=<seed>]                                     An optional seed used in hashing for sampling cohort selection
+        |  [--hashAlgorithm=(murmur|farm)]                     An optional arg to select the hashing algorithm for sampling cohort selection. Defaults to FarmHash for BigQuery compatibility
+        |  [--distribution=(uniform|stratified)]               An optional arg to sample for a stratified or uniform distribution. Must provide `distributionFields`
+        |  [--distributionFields=<field1,field2,...>]          An optional list of fields to sample for distribution. Must provide `distribution`
+        |  [--exact]                                           An optional arg for higher precision distribution sampling.
+        |  [--byteEncoding=(raw|hex|base64)]                   An optional arg for how to encode fields of type bytes: raw bytes, hex encoded string, or base64 encoded string. Default is to hash raw bytes.
```

Comment:
Nit: Is it possible to separate formatting changes into a different PR?

Reply:
not really, since I'm aligning the new whitespace with the field being added in this PR